dist/style.scss

```diff
@@ -468,6 +468,126 @@ line-height: 24px;
   }
 }
 
+#contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  .contact-text {
+    display: flex;
+    // align-items: center;
+    // border: 3px solid red;
+    width: 327px;
+    height: 216px;
+    margin: 5rem 1rem 3rem 0;
+    p {
+      font-family: $inter;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 32px;
+      line-height: 44px;
+      color: $base-navy;
+    }
+  }
+}
+
+.form {
+  margin-top: 10rem;
+  fieldset {
+    width: 384px;
+    height: 350px;
+    position: relative;
+    border: none;
+    .input-group {
+      width: 100%;
+      input {
+        height: 35px;
+      }
+
+      input,
+      textarea {
+        margin-left: 1rem;
+        width: 100%;
+        border: none;
+        border-bottom: 1px solid $gray3;
+      }
+
+      input:focus,
+      textarea:focus {
+        outline: none;
+      }
+
+      textarea {
+        background-color: #e7e5e5;
+      }
+
+      .submitErr {
+        color: red;
+      }
+      .fa-sharp {
+        color: green;
+      }
+
+      .firstNerror {
+        position: absolute;
+        right: 360px;
+        top: 10px;
+      }
+
+      .lastNerror {
+        position: absolute;
+        right: 15px;
+        top: 10px;
+      }
+
+      .fullNerror {
+        position: absolute;
+        right: 20px;
+        top: 10px;
+      }
+
+      .emailError {
+        position: absolute;
+        right: 20px;
+        top: 45px;
+      }
+
+      .msgError {
+        position: absolute;
+        right: 20px;
+        top: 200px;
+      }
+
+      .fullN,
+      #fullName {
+        display: block;
+      }
+
+      .err {
+        color: red;
+      }
+    }
+    .allNames {
+      display: none;
+    }
+    button {
+      margin-top: 1.5rem;
+      width: 122px;
+      height: 48px;
+      background-color: $primary;
+      color: $base-white;
+      border: none;
+      font-family: $roboto;
+      font-style: normal;
+      font-weight: weight(bold);
+      font-size: 17px;
+      line-height: 24px;
+      margin-left: 1rem;
+    }
+  }
+}
```

Comment:
- [ ] I think there are a couple of things that can be improved about your contact form to make it as close to the [FIGMA template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as possible.

    - It would be better to improve the input field design to match that of Figma :+1:  you can kindly use ```padding:``` and ```margin``` to fix this issue.
    - It will be better too to use the same text formatting as that of Figma 
![image](https://user-images.githubusercontent.com/102757126/199197011-2933e222-86af-4daa-bf2b-2b3faf711082.png)
    - It would be better to use ```background-color: #fbf8f7;``` on the comment section to make it similar to that of Figma design :joy: 
    

**Your design**
![image](https://user-images.githubusercontent.com/102757126/199194698-e1908bca-75be-45f6-865e-8dde3149000b.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/199194880-9dd3a1ae-5272-4c40-b66b-ecbcd14add1e.png)